public/index.html

```diff
@@ -30,7 +30,7 @@
 
     <script type="text/javascript">
       window.config = {
-        domainName: "http://localhost:1880",
+        domainName: "http://reactadmin-on-nodered.ripple.foundation:1880",
```

Comment:
this should not be hard coded like this

Reply:
You also need to explain how to fire up the application , which URL to use? its not localhost:1880

Reply:
Now I left the empty value. In ReadMe file I added info, that it is necessary to define domain name in index.html.